lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -305,6 +322,55 @@ public void useHardwareAcceleration(boolean use) {
   }
 
   /**
+   * Sets the animation from a file in the raw directory.
+   * This will load and deserialize the file asynchronously.
+   * <p>
+   * Will not cache the composition once loaded.
+   */
+  public void setAnimation(int animationResId) {
```

Comment:
`@RawRes`

Reply:
I looked at the rest of the class and it's not using support annotations. For instance the signature for `setImageResource` is `@Override public void setImageResource(int resId)`.

But I'll change that.

Reply:
@cyrilmottier It may be missing in a few cases but the intention is for them to be there. Feel free to add any you find that are missing!